Assets/Scripts/Views/AssetViewer.cs

```diff
@@ -100,4 +118,55 @@ public void Lighten(GameObject prefab)
     {
         prefab.GetComponentInChildren<Image>().color = Color.white;
     }
+
+    public void MoveTo(GameObject prefab, Vector2 target, float speed, MovementTypes style)
+    {
+        moveCoroutine = StartCoroutine(MoveAsset(prefab, target, speed, style));
+    }
+    
+    //For SmoothDamp only; higher speed = slower movement
+    private IEnumerator MoveAsset(GameObject prefab, Vector2 target, float speed, MovementTypes style)
+    {
+        isMoving = true;
+        movingPrefab = prefab;
+        targetPosition = target;
+        
+        Vector2 currentPosition = prefab.transform.position;
+        Vector2 velocity = Vector2.zero;
+
+        speed *= Time.deltaTime;
+
+        while (Vector2.Distance(currentPosition, target) > .1f)
+        {
+            switch (style)
+            {
+                case MovementTypes.Smooth:
+                    currentPosition = Vector2.MoveTowards(currentPosition, target, speed * 50);
+                    break;
+                case MovementTypes.FastStart:
+                    currentPosition = Vector2.Lerp(currentPosition, target, speed);
+                    break;
+                case MovementTypes.FastMiddle:
+                    currentPosition = Vector2.SmoothDamp(currentPosition, target, ref velocity, speed * 10);
+                    break;
+            }
+            
+            PlaceAt(prefab, currentPosition);
+            yield return new WaitForSeconds(.01f);
```

Comment:
Having this at `.01f` makes one of the tests time out, and considerably slows down the animations as a whole. Changing this to `.001f` makes it on par with one of the earlier commits.

Reply:
I retested .01f on my end and the test doesn't time out in Unity, but I do agree that it's slower than it previously was. I'll change it to .001f and we'll see if that fixes the autotesting.